docs/wip/data-fetching.md

```diff
@@ -1,8 +1,45 @@
-## Gatsby config queries
-
 ## How to fetch data for your pages
 
+One of the main benefits of Gatsby is its hybrid nature capable of static and dynamic data fetching, which means that is possible to fetch data in build time as well in runtime on client-side. The choice of which approach to use depends on the nature of the data and its context, in the section below we will explain the difference with examples.
+
+> If you want to read more about this topic, check the [Data Fetching](https://www.gatsbyjs.com/docs/data-fetching/) Gatsby documentation.
+
 ### Static data
+
+Static data is the content that does not change frequently or per user interaction. For example, a product's page can be a data retrieve static since the description and attributes of product don't change very often. This means that this data can be fetch during build time and when the user access the products page the data is already loaded.
```

Comment:
"`Static data` is the content that does not change frequently or per user interaction. 
Considering that, you can use the fecthing during building time for data that don't change very often, making the storefront loading faster. 

For example, a product's page can be a data retrieve static since the description and attributes of a product rarely chages. This means that this data can be fetch during build time and when the user access the products page the data is already loaded."

the sugestion is to put the conclusion before the exemple, so we can be straight to the point. Also, smaller text lines